docs/concepts/SafeRollout/README.md

```diff
@@ -195,6 +195,25 @@ For `Service` based on the service type the availability is determined as follow
 - For `ExternalName` service, checking availability is not supported, so it will be marked as available with a "not trackable" reason.
 
 
+#### Custom Resource Definition
+For `CustomResourceDefinition` availability is determined as follows:
+- Condition `Established` is set to true.

```

Comment:
Are these two conditions an "AND" relation or "OR" relation? Other cases are all "OR" relation.